wiki/Client/Options/de.md

```diff
@@ -1,292 +1,431 @@
+| `Private Nachrichten nur von Freunden` | Sofern aktiviert, werden nur private Nachrichten von Freunden akzeptiert. | Checkbox | `Deaktiviert` |
+| `Ignorierliste (leerzeichengetrennt)` | Wörter, die du hier platzierst, werden ignoriert. | Text | *(leer)* |
+| `Highlight-Wörter (leerzeichengetrennt)` | Wörter, die du hier platzierst, werden im Chat hervorgehoben. | Text | *(leer)* |
+
+## Wartung
+
+![Symbol für die Sektion Wartung](img/maintenance-DE.jpg "Symbol für die Sektion Wartung")
+
+Dieser Abschnitt ist über Beatmaps und Aktualisierungen.
+
+### Allgemein
+
+| Name | Beschreibung | Typ |
+| :-- | :-- | :-- |
+| `Alle nicht gerankten Maps löschen` | Lösche alle nicht gerankten Maps aus dem Song-Ordner. | Button |
+| `Ordnerberechtigungen erzwingen` | Erzwingt Lese-/Schreib-Berechtigungen für den Spiel-Client, um auf die entsprechenden Ordner zugreifen zu können. (Damit das funktioniert, wird das Administrator-Passwort benötigt.) | Button |
+| `Alle Maps als "Gespielt" markieren` | Markiert alle Maps als "Gespielt". | Button |
+| `osu!-Updater starten` | Schließt den Spiel-Client und öffnet den Updater, um nach Aktualisierungen zu suchen und herunterzuladen, sofern verfügbar. | Button |
+
+Weiter unten wird die Versionsnummer angezeigt. Bei einem Klick wird das [Änderungsprotokoll](/wiki/Glossary/Changelog) für diese Version in deinem Webbrowser geöffnet.
+
+### Debug
+
+#### Build-Version
+
+Schaue, welche Build-Version dein Spiel-Client aktuell hat und welche Updatequelle dein Spiel-Client nutzt. Bei einem Klick wirst du zum Änderungsprotokoll in deinem bevorzugten Webbrowser weitergeleitet.
+
+Die Builds werden nach folgendem Schema nummeriert:
+
+```
+b{YYYY}{MM}{DD}.{revision}{type}
+```
+
+- `{YYYY}` ist das Jahr des Builds
+- `{MM}` ist der Monat des Builds
+- `{DD}` ist der Tag des Builds
+- `{revision}` ist die Überarbeitung des Builds
+  - Wenn keine Überarbeitung angegeben ist, wird der Dezimalpunkt entfernt.
+- `{type}` ist die Updatequelle
+  - Wenn kein Wert hier angegeben ist, nehme an, dass es `Stable` ist.
+
+## Trivia
+
+- Wenn du einen Benutzernamen eingibst, aber das Passwortfeld leer lässt, wird osu! diesen Namen benutzen, falls Scores lokal gespeichert werden.
+- Das Öffnen der Options-Leiste wird automatisch die Funktion `osu! is up-to-date!` aktivieren (es wird geprüft, ob Aktualisierungen verfügbar sind).
+- Die Option `Saisonale Hintergründe` wurde hinzugefügt, nachdem das osu!dev-Team positives Feedback erhalten hatte.
+  - Mehr Details: [main menu background changes](https://osu.ppy.sh/community/forums/topics/606931)
+
+### Geschichte
 
-- b(Jahr)(Monat)(Tag).(Überarbeitung) = Public Build. Also, b20140323.3 bedeutet: Public Build vom 23. März 2014, dritte Überarbeitung.
+- Das alte Options-Fenster war tatsächlich ein Fenster, das Tabs, Buttons und einen dunklen hellblauen Hintergrund hatte.
```

Comment:
```suggestion
- Das alte Options-Fenster war tatsächlich ein Fenster, das Tabs, Buttons und einen dunklen blassblauen Hintergrund hatte.
```
`dunklen hellblauen` sounds contradictory

Reply:
Unrelevant information: This was a suggestion by deepl.